python/lsst/dax/obscore/cli/cmd/commands.py

```diff
@@ -30,10 +32,14 @@
     repo_argument,
     where_option,
 )
-from lsst.daf.butler.cli.utils import ButlerCommand, MWPath, split_commas
+from lsst.daf.butler.cli.utils import ButlerCommand, MWPath
 
 from ... import script
 
+split_commas = cast(
+    Callable[[click.Context, click.Option | click.Parameter, Any], Any], cmd_utils.split_commas
```

Comment:
Remind me why we can't fix the annotation in butler?

Reply:
I don't remember now, but I can try to fix it.

Reply:
Looks like mypy 1.1.1 does not even need that cast (or maybe it's newer click). It complained about split_commas at USDF, but at my local machine with latest everything it looks fine.

Reply:
USDF has types-click installed which breaks everything (we are trying to remove it from rubin-env-developer).

Reply:
So, if I remove those casts (there are also similar casts in daf_butler) people developing at USDF will suffer? Should I keep them for now?

Reply:
I think that we are the main mypy users of daf_butler click tooling so I think it's fine to delete them (and other packages will break if types-click is present). @ktlim should be removing types-click in the next release.

Reply:
OK, I'll remove casts from dax_obscore and daf_butler.